.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,7 +11,8 @@
 <!-- Explain, without going into the details, what this PR does, and what problem it solves. -->
 
 **Which issue(s) this PR fixes**:
-<!-- Add a link to the JIRA issue. Otherwise, put "no issue."
+<!-- Add a link to the JIRA issue. Otherwise, put "no issue." 
+If this solves a COPS issue, you must include release notes below.
```

Comment:
My suggestion would be to move this sentence to the "Does this PR introduce a user-facing change?" section and re-phrase it like this:

```
If the change fixes a COPS issue, you must include a relevant release note here.
```